Project.toml

```
@@ -1,13 +1,14 @@
 name = "LicenseCheck"
 uuid = "726dbf0d-6eb6-41af-b36c-cd770e0f00cc"
 authors = ["Eric P. Hanson"]
-version = "0.2.2"
+version = "0.3.0"
```

I don't think this is actually breaking, it's just new features
```suggestion
version = "0.2.3"
```